main.go

```diff
@@ -413,7 +416,15 @@ func decodeGMO(ctx context.Context, sDec []byte, username string) string {
 			if fort.ActivePokemon != nil {
 				newMapPokemon = append(newMapPokemon, decoder.RawMapPokemonData{Cell: mapCell.S2CellId, Data: fort.ActivePokemon})
 			}
+			// collect fort information of cell
+			switch fort.FortType {
+			case pogo.FortType_CHECKPOINT:
+				stopIdsPerCell[mapCell.S2CellId] = append(stopIdsPerCell[mapCell.S2CellId], fort.FortId)
```

Comment:
is it possible to collect that information in that way @jfberry ?

```go
var stopIdsPerCell = make(map[uint64][]string)
```
appending the map value with content, or is it needed to initialize the slices? :)